### DISCUSSION TITLE
Welcome to DigiByte-Pipeline Discussions!

### DISCUSSION THREAD

**Discussion by gto90** [DigiByte-Core/DigiByte-Pipeline#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DGBNOOB** [comment]
Greeting GTO,

I'm Jose, I found DigiByte in July 2021 and I've been fascinated by DigiByte ever since. Thank you for adding the discussion, issues and Gitter features to the DigiByte pipeline and I hope we can get more community involvement on Github even if it's as non-code contributors. I often like to use the example of being on an assembly line and everyday we add something new to our understanding and skill set. I will try to encourage some of the other community members to come join the conversation. 

-Jose DGB Noob

**Reply by gto90** [reply]
You are most welcome Jose!  If there is anything else I can do please let me know!
